app/src/main/AndroidManifest.xml

```diff
@@ -1,22 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="io.intrepid.skeleton"
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.intrepid.skeleton"
     >
 
     <application
+        android:name=".SkeletonApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         >
-        <activity android:name=".MainActivity">
+        <activity android:name=".screens.example1.Example1Activity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
 
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <activity android:name=".screens.example2.Example2Activity">
+        </activity>
```

Comment:
Nit: empty tags should use the short version (`<activity attr="foo" />`) with the close tag on a new line.